script.js

```diff
@@ -1,11 +1,91 @@
+$(document).ready(function(){
+
 // Create an array called `colors` that contains three colors.
 
+
+
+var colors = ['red', 'green', 'yellow'];
+var instructor = {
+	name: "Whitney",
+	expertise: "CSS",
+	location: "Washington, D.C.",
+};
+
 // Log the last item in the array to the console.
 
+console.log(colors[2]);
+
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
 
+for (var i = 0; i < colors.length; i++) {
+	console.log("My favorite color is [i]");
```

Comment:
Inside of plain strings, variable names just look like any other character and will be printed as-is.

```js
console.log("My favorite color is [i]");
// => "My favorite color is [i]"
```

To add the contents of a variable to the string we need to use a technique like concatenation `+`.

```js
var fave = "brown";
console.log("My favorite color is " + fave);
```

Also (outside of a string) when JS sees square brackets without an array name `[i]`  JS will treat it like an array literal and create an array with the sole element `i` in this case. When getting a value from an array by index, remember to specify the array name.

```js
console.log("My favorite color is " + colors[i]);
```